script.js

```diff
@@ -184,40 +184,26 @@ const Cards = [
 
 //---------------Arrays---------------//
 
-// decks & wastes
-let playerDeck = [];
-let opponentDeck = [];
-let playerWaste = [];
-let opponentWaste = [];
+// decks
+let blueDeck = [];
+let redDeck = [];
 
 // hands
-let playerHand = [];
-let opponentHand = [];
-
-// Board elements
-let mainBoard = [];
-let opponentClaimedBoard = [];
-let playerClaimedBoard = [];
+let blueHand = [];
+let redHand = [];
 
 // build table
-const table = [];
-table['pd'] = playerDeck;
-table['ph'] = playerHand;
-table['pw'] = playerWaste;
-table['pc'] = playerClaimedBoard;
-table['od'] = opponentDeck;
-table['oh'] = opponentHand;
-table['ow'] = opponentWaste;
-table['oc'] = opponentClaimedBoard;
-table['main'] = mainBoard;
+const hands = [];
+hands['bh'] = blueHand;
+hands['rh'] = redHand;
 
 //---------------Variables---------------//
 
 // declare variables
 const d = document;
 const handCount = 5;
 const deckSize = 20;
```

Comment:
```suggestion
const d = document;
const HAND_COUNT = 5;
const DECK_SIZE = 20;
```
When declaring an actual constant that won't change, it's common practice to use all caps (aka SCREAMING_SNAKE_CASE). The idea lies in signaling to other devs that this value will not change. I'd probably move the constants to the very top of the script. IMO this will structure the script better and increases readability

Reply:
Though, the `d` shorthand variable seems fine as-is^^

Reply:
Got it, makes total sense.